website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -96,6 +96,16 @@ timestamp appended.
 
 ## Nomad 1.0.10 and 1.1.4
 
+#### Strict mTLS certificate check
+
+Raft requests were not being verified for their certificate common alternative
+name. A check was introduced to make sure only servers in the same region are
+able to make Raft requests. This additional check results in a breaking change
+where wildcard values in the certificate alternative name list are not
+accepted.
+
+This breaking change will be fixed in a future version of Nomad.
```

Comment:
A follow-up PR will update this line with the proper version where this was fixed.

Reply:
Let's also add:

> It is recommended to avoid wildcards and use different certificates for clients and servers.